api/src/main/java/io/strimzi/api/kafka/model/connect/KafkaConnectResources.java

```diff
@@ -117,6 +117,30 @@ public static String initContainerClusterRoleBindingName(String clusterName, Str
         return "strimzi-" + namespace + "-" + componentName(clusterName) + "-init";
     }
 
+    /**
+     * Get the name of the Kafka Connect role binding given the name of the {@code cluster}.
+     *
+     * @param clusterName  The cluster name.
+     *
+     * @return The name of Kafka Connect role binding.
+     */
+    public static String connectRoleBindingName(String clusterName) {
+        return componentName(clusterName) + "-connect-role";
+    }
+
+    /**
+     * Get the name of the internal secret that contains TLS trusted certificates.
+     * The operator copies user specified secrets for trusted certificates into
+     * a single secret with this name. It is then used when configuring Connect.
+     *
+     * @param clusterName The cluster name.
+     *
+     * @return Name of the internal secret that contains TLS trusted certificates.
+     */
+    public static String internalTlsCertsSecretName(String clusterName) {
+        return componentName(clusterName) + "-tls-certs";
+    }
```

Comment:
just having TLS in both method and Secret name seems too generic. Would it be better to have also "trusted" somewhere? i.e. `internalTlsTrustedCertsSecretName` and "-tls-trusted-certs" suffix?